docs/user_manual.md

```diff
@@ -189,6 +189,12 @@ docker.resizeTty("containerID", height, width);
 docker.startContainer("containerID");
 ```
 
+### Stop a container
+
+```java
+docker.stopContainer("", 10); // kill after 10 seconds
```

Comment:
Instead of empty string, we should indicate the first param needs to be a container ID.